### PR TITLE
App can successfully like a restaurant 

### PR DESCRIPTION
**Pull request by bryanfong-dev-old**
- app can like restaurant 
- deleted old yelp api text file 
- fixed merge conflicts
- uncommented isAuthenticated

### CODE REVIEW

File: server/controllers/dbController.js [hunk 1]
```diff
@@ -41,19 +41,20 @@ dbController.addRestaurant = (req, res, next) => {
...
-      console.log('result : ', result);
       console.log(`successfully added ${name} to db`);
       res.locals.rest_id = result.rows[0]._id;
       return next();
     })
-    .catch( (err) => {
-      console.log("error after liking",err);
+    .catch((err) => {
+      console.log("error trying to add to database", err);
       res.status(400).send(err)
```
Comment (conor-jh) [inline server/controllers/dbController.js]: Change to `500` status

**Review by conor-jh — CHANGES_REQUESTED** [review]
Looks good after we update that status code.

**Review by Donaldblanc — APPROVED** [review]
remove the comments but otherwise looks good.

**Review by conor-jh — APPROVED** [review]
Sweet 👍

**Pull request merged**